README.md

```diff
@@ -119,6 +119,7 @@ See the [Docker example folder](https://github.com/NVIDIA/apex/tree/master/examp
 * Python 3.6
 * Pytorch 1.5 or newer, The HIPExtensions require 1.5 or newer.
 * We recommend follow the instructions from [ROCm-Pytorch](https://github.com/ROCmSoftwarePlatform/pytorch) to install pytorch on ROCm.
+* Note: For Pytorch versions less than 1.8 please consider release package [ROCm-Apex v0.3](https://github.com/ROCmSoftwarePlatform/apex/releases/tag/v0.3) . For pytorch versions >= 1.8, use the master branch as this contains latest hipify revamp changes.
```

Comment:
Should we reword this to give a stronger guidance that building ROCm apex from source is *not supported* anymore for pytorch < 1.8? That's my understanding, please correct me if I'm wrong.

Reply:
We can do that. I would let @sunway513 or @jeffdaily to comment on this one.

Reply:
Keep the suggestion to use a release package for pytorch versions < 1.8.  Add the stronger guidance that building rocm apex from source is not supported for pytorch versions < 1.8.

Reply:
Thanks @jeffdaily. @lcskrishna Once you make this change and the other nit change, we should merge this in. I'll need it to get the DeepSpeed CI back on track.

Reply:
Done